components/Chart.tsx

```diff
@@ -147,6 +151,14 @@ const ExpandableBarChartByPattern = (props: ExpandableBarChartByPatternProps) =>
   const [open, setOpen] = useState(false)
   const [openIndex, setOpenIndex] = useState(0)
 
+  const handleClick = (i: number) => {
+    if (open) {
+      setOpenIndex(i);
+    } else {
+      setOpen(!open);
+    }
+  }
```

Comment:
this also fixes a glitch where you had a close a previous bar before opening another, now we'll just switch which is open in a single click